workers/vita/Dockerfile

```diff
@@ -17,56 +17,77 @@ RUN sed 's/^deb \(.*\)/deb-src \1/' /etc/apt/sources.list \
 ENV VITASDK=/opt/vitasdk
 ENV PATH=$PATH:$VITASDK/bin
 
-ARG URL=https://github.com/vitasdk/autobuilds/releases/download/master-linux-v567/vitasdk-x86_64-linux-gnu-2017-10-29_09-48-44.tar.bz2
+#Install VitaSDK base
+ARG URL=https://github.com/vitasdk/autobuilds/releases/download/master-linux-v611/vitasdk-x86_64-linux-gnu-2017-12-01_14-01-36.tar.bz2
 
 RUN wget --progress=bar:force:noscroll -O - $URL \
 	| tar --no-same-owner --no-same-permissions -C /opt/ -xjf -
 
-# These insecure, unversioned downloads are not great, but OpenSSL and SDL2 are
-# not trivially buildable for this target so we use these packages for the
-# moment. At the time of the first build these corresponded to
-# https://github.com/vitasdk/packages/commit/b62cb68c349f8f389a4b4e57356c398c43f5531d
-# in case someone needs to rebuild them from source later
-RUN wget --progress=bar:force:noscroll -O - http://dl.vitasdk.org/openssl.tar.xz \
-	| tar --no-same-owner --no-same-permissions -C $VITASDK/arm-vita-eabi -xJf -
-RUN wget --progress=bar:force:noscroll -O - http://dl.vitasdk.org/curl.tar.xz \
-	| tar --no-same-owner --no-same-permissions -C $VITASDK/arm-vita-eabi -xJf -
-RUN wget --progress=bar:force:noscroll -O - http://dl.vitasdk.org/sdl2.tar.xz \
-	| tar --no-same-owner --no-same-permissions -C $VITASDK/arm-vita-eabi -xJf -
-RUN wget --progress=bar:force:noscroll -O - http://dl.vitasdk.org/sdl_net.tar.xz \
-	| tar --no-same-owner --no-same-permissions -C $VITASDK/arm-vita-eabi -xJf -
-
+#Install git
+RUN apt-get -y install git
+
+#Install sudo
+RUN apt-get -y install sudo
```

Comment:
Docker containers are not operating systems. Sudo is not an appropriate thing to be used in a container.